### DISCUSSION TITLE
State of v2.6

### DISCUSSION THREAD

**Discussion by NayamAmarshe** [upscayl/upscayl#384] [Announcements]
So the next version of Upscayl is going to be v2.6 which is going to bring general bug fixes and some extra tools like image manipulation (compression and filters).

I'll be a bit busy for the next 2 weeks though, so the development is going to be paused till then BUT, I'll try my best to release v2.6 by the end of this month or the first 2 weeks of August :)

Thanks!

**Comment by NayamAmarshe** [comment]
The work has started on v2.6/2.7.5

Just a few things left :)

**Comment by XGabrielX007** [comment]
Hello Friend, why did VirusTotal detect this? 

  https://www.virustotal.com/gui/file/07e49f7cbb4ede01ae4dd4c399d3a7e5846e3d2085c3128eff881e55cb7b1a0c/community

![image](https://github.com/upscayl/upscayl/assets/143371282/92be49f1-9d84-45ea-a07e-a257f7a06010)

**Reply by begin-theadventure** [reply]
1/71, so obviously a false positive, and your question is completely unrelated to v2.6, which hasn't even been released yet.

**Reply by aaronliu0130** [reply]
Yeah, that file's 1 for 1 https://github.com/xinntao/Real-ESRGAN-ncnn-vulkan/releases

**Reply by XGabrielX007** [reply]
@begin-theadventure sorry friend, I created my account 2 days here github, I'm learning to mess with
